<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kapudan</class>
 <widget class="QWidget" name="kapudan">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>751</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kapudan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/kapudan.qrc">
    <normaloff>:/raw/pixmap/cr64-app-kapudan.png</normaloff>:/raw/pixmap/cr64-app-kapudan.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#kapudanContainer{
	background-image: url(:/raw/pixmap/bg2.png);
   background-color: rgb(26, 29, 32);
	padding: 0px;
	margin: 0px;
}

#description {
	color:rgb(240, 240, 240);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="kapudanContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="screenTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0px;
background-color: rgba(0,0,0,0);
border-bottom: 1px solid rgba(0,0,0,10);
color:rgb(240, 240, 240);
qproperty-alignment: AlignCenter;</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>Welcome to Chakra</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>129</width>
            <height>50</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 10px 0px 10px 0px; /*13px 15px 13px 15px;*/
border:0px;
background-color: rgba(0,0,0,0);
border-bottom: 1px solid rgba(0,0,0,10)</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelProgress">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
background-color: rgba(0, 0, 0, 70);
</string>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMenu">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
background-color: rgba(0, 0, 0, 70);</string>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMenu_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
background-color: rgba(0, 0, 0, 70);
padding: 10px;
</string>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="mainStack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QStackedWidget#mainStack{
margin: 0px;
border-radius: 0px;
color: white;
}
QLineEdit {
border: 1px solid gray;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxMenu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>33</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#groupBoxMenu{
    background-color: rgba(0, 0, 0, 70);
}


QPushButton{
	padding: 3px 10px 3px 10px;
	color: white;
	border:0px;
	border-radius:4px;
	background-color: rgb(68, 69, 70); /*rgba(0, 0, 0, 70);*/
}

QPushButton::flat{
	border:1px solid  rgb(26, 29, 32); /*rgba(0, 0, 0, 70);*/
	border-radius:4px;
	background-color: rgb(68, 69, 70); /*rgba(0, 0, 0, 70);*/
}

QPushButton::flat:hover{
	border:1px solid rgba(0, 0, 0, 100);
	border-radius:4px;
	background-color: rgba(0, 172, 255);
}

QPushButton::flat:pressed{
	border:1px solid rgba(0, 0, 0, 100);
	border-radius:4px;
	background-color: rgba(106, 114, 128);
}



/*
QPushButton::flat:pressed{
	border:1px solid  rgba(34, 49, 60, 100);
	border-radius:2px;
	padding: 3px 10px 3px 10px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(3, 3, 129, 255), stop:0.0192308 rgba(25, 35, 160, 255), stop:0.521739 rgba(29, 35, 166, 255), stop:0.531585 rgba(40, 46, 184, 255), stop:0.983696 rgba(74, 78, 168, 255), stop:1 rgba(110, 110, 210, 255));
}
*/

/*
QPushButton{
	padding: 3px 10px 3px 10px;
	color: white;
	border:0px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(132, 132, 132, 255), stop:0.501481 rgba(70, 70, 70, 255), stop:0.503057 rgba(62, 62, 62, 255), stop:1 rgba(16, 16, 16, 255));
}

QPushButton::flat:hover{
	border:1px solid  rgba(34, 49, 60, 100);
	border-radius:2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgb(35, 80, 153), stop:1 rgba(50, 173, 241, 255));
}

QPushButton::flat{
	border:1px solid  rgba(34, 49, 60, 100);
	border-radius:2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(132, 132, 132, 255), stop:0.501481 rgba(70, 70, 70, 255), stop:0.503057 rgba(62, 62, 62, 255), stop:1 rgba(16, 16, 16, 255));
}

QPushButton::flat:pressed{
	border:1px solid  rgba(34, 49, 60, 100);
	border-radius:2px;
	padding: 3px 10px 3px 10px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(3, 3, 129, 255), stop:0.0192308 rgba(25, 35, 160, 255), stop:0.521739 rgba(29, 35, 166, 255), stop:0.531585 rgba(40, 46, 184, 255), stop:0.983696 rgba(74, 78, 168, 255), stop:1 rgba(110, 110, 210, 255));
}
*/</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="buttonCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="../data/kapudan.qrc">
             <normaloff>:/raw/pixmap/cross.png</normaloff>:/raw/pixmap/cross.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBack">
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset resource="../data/kapudan.qrc">
             <normaloff>:/raw/pixmap/arrow-left.png</normaloff>:/raw/pixmap/arrow-left.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonNext">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
           <property name="icon">
            <iconset resource="../data/kapudan.qrc">
             <normaloff>:/raw/pixmap/arrow-right.png</normaloff>:/raw/pixmap/arrow-right.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonApply">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="icon">
            <iconset resource="../data/kapudan.qrc">
             <normaloff>:/raw/pixmap/arrow-right.png</normaloff>:/raw/pixmap/arrow-right.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonFinish">
           <property name="text">
            <string>Finish</string>
           </property>
           <property name="icon">
            <iconset resource="../data/kapudan.qrc">
             <normaloff>:/raw/pixmap/tick.png</normaloff>:/raw/pixmap/tick.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/kapudan.qrc"/>
 </resources>
 <connections/>
</ui>
